DependencyInjection/Configuration.php

```diff
@@ -273,18 +277,13 @@ private function addFormatListenerSection(ArrayNodeDefinition $rootNode)
                             ->end()
                         ->end()
                         ->arrayNode('media_type')
-                            ->children()
-                                ->arrayNode('version_regex')
-                                    ->canBeEnabled()
-                                    ->beforeNormalization()
-                                        ->ifString()->then(function($v) { return array('enabled' => true, 'regex' => $v); })
-                                    ->end()
-                                    ->children()
-                                        ->scalarNode('service')->defaultNull()->end()
-                                        ->scalarNode('regex')->defaultValue('/(v|version)=(?P<version>[0-9\.]+)/')->end()
-                                    ->end()
-                                ->end()
-                            ->end()
+                            ->canBeEnabled()
+                            ->beforeNormalization()
+                            ->ifString()->then(function($v) { return array('enabled' => true, 'version_regex' => $v); })
```

Comment:
and the wrong indentation means you are missing a ``->end()`` call, which might hurt you in the future (the ``->end()`` call at the same indentation than ``->arrayNode('media_type')`` is not ending this array node anymore)